README.md

```diff
@@ -83,7 +83,8 @@ $ npm run electron-rebuild # You MAY need to run this if MagicMirror complains a
        "clientSecret": "YOUR_CLIENT_SECRET",
        "deviceId": "YOUR_DEVICE_ID",
        "refreshToken": "YOUR_REFRESH_TOKEN",
-       "lite": false
+       "lite": false,
+       "visualization": false
```

Comment:
I would make this more explicit name like: `isSpeechVisualizationEnabled` etc :)